google-cloud-testing/storage-benchwrapper/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>benchwrapper</artifactId>
+  <version>0.0.0</version>
+  <packaging>jar</packaging>
+  <groupId>com.google.cloud</groupId>
+
```

Comment:
Without this module being added to https://github.com/googleapis/google-cloud-java/blob/82ab4cb829b1121052689b7f4de738e54e1ad68d/google-cloud-testing/pom.xml#L21-30 it won't actually be loaded and ran as part of the build. Is that intentional?

Reply:
Yes.